app/routes/metrics.py

```diff
@@ -1,9 +1,15 @@
-from typing import Annotated, Literal, List
+from typing import Annotated, Literal, List, Union
```

Comment:
Change this import (see "Importación" in "Acuerdos de Estilo" for the [agreements in python projects](https://docs.google.com/document/d/1PQOE35s5OkW3N5B9V5ql27__btKW6qyvl6vRvK7yG_w/edit)